modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-transfer-leadership.adoc

```diff
@@ -0,0 +1,53 @@
+= rpk cluster partitions transfer-leadership
+
+Transfer partition leadership between brokers.
+
+This command allows you to transfer partition leadership.
```

Comment:
These lines pretty much say the same thing. Let's consolidate into one statement:
Allows you to transfer partition leadership between brokers.

Reply:
Removed one of them.